### ISSUE TITLE
Feature request: Implement my anti-cheat solution

### ISSUE THREAD

**Issue by Slluxx** [spacemeowx2/slp-server-rust#27]
I have written an anticheat sysmodule and would like to talk with you about a lan-play-server integration. Im not familiar with rust so im struggeling to implement this myself but thought this might be something, some servers would like to have.

Basically lan-play recieving the data by my sysmodule and then blocking traffic, preventing them to join any game on this server.

Are you reachable on discord for some private discussions about this topic? My discord handle is `Snyv#7598`

**Comment by spacemeowx2** [comment]
Quick question: How does your anti-cheat solution work? Is it a simple blocklist or packet based analyzer?

**Comment by Slluxx** [comment]
Currently it sends the information i need to my server, which then makes the data [publicly available](https://vue-w3otbc.stackblitz.io/) for server owners (the demo data will be gone once someone actually plays). I hope that i can send this data to the lan-play-server instead and based on that, it decides to block or allow traffic. 

How exactly it works is not something i want to discuss publicly right now.

**Comment by spacemeowx2** [comment]
So I think it has contains 3 feature requests:

1. `slp-server-rust` sends traffic to your service.
2. Add an API that receive control commands to block or allow clients' traffic.
3. Only allow clients with anti-cheat sysmodule if needed.

**Comment by Slluxx** [comment]
Not quite. If possible, i want to completely decouple my service from this. I just made the service as a way to evaluate players before it can actually be implemented in the server. The data the sysmodule sends, should be picked up by the server and then get evaluated. This way, there are no external requirements.

**Comment by spacemeowx2** [comment]
Sounds good, so in lan-play server I should implement a middleware
